src/gtc/common.py

```diff
@@ -317,6 +317,9 @@ class Literal(Node):
     dtype: DataType
     kind: ExprKind = ExprKind.SCALAR
 
+    def __repr__(self) -> str:
+        return str(self.value)
```

Comment:
Just a question: Where is this used?

Reply:
I can't tell 😆 . I'll try disabling it run the tests.